client/sprotty-ecore/src/views.tsx

```diff
@@ -131,15 +131,18 @@ export class AggregationEdgeView extends DiamondEdgeView {
 @injectable()
 export class LabelNodeView extends SLabelView {
   render(labelNode: Readonly<SLabelNode>, context: RenderingContext): VNode {
-    const image = require("../images/" + labelNode.imageName);
+      let image;
+      if (labelNode.imageName) {
+          image = require("../images/" + labelNode.imageName);
```

Comment:
can't we somehow use a `import` instead of the `require`? I didn't find anything useful in a quick search though,

Reply:
We could do it with the async [import statement](https://v8.dev/features/dynamic-import)
Though this would complicate things a little bit and impede readability.

Why would you prefer import over require?

Reply:
import is part of the standard and require is commonjs specific

Reply:
Ah okay :) 
well, the biggest disadvantage is, that import statement allows a "dynamic" module name only in async calls. But sure if you'd rather have this done with import it is also possible, I think

Reply:
I think we can merge it with the required statement and discuss `import` vs `required` later